perfkitbenchmarker/linux_benchmarks/ping_benchmark.py

```diff
@@ -25,10 +25,6 @@
 from perfkitbenchmarker import vm_util
 import re
 
-flags.DEFINE_boolean('ping_also_run_using_external_ip', False,
```

Comment:
Looks like @miketruty 's new tutorials used this flag.  
https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/d486aa9ba587c44dc3edc43f75b8fa0d7222f1f7/tutorials/inter_region_reports/data/simple_region_latency.yaml#L16

Are they working currently given that we don't have the firewalls in place?  
Preferences on updating these all together or seperately?

Reply:
There is currently an error when using external IPs in the tutorial, which is what originally prompted this PR. I'll also update the tutorial to use the new stuff